pkg/desktop/desktop_platform_linux.go

```diff
@@ -0,0 +1,43 @@
+//go:build linux
+// +build linux
+
+package desktop
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/0xAX/notificator"
+	"github.com/stashapp/stash/pkg/logger"
+)
+
+// isService checks if started by init, e.g. stash is a *nix systemd service
+func isService() bool {
+	return os.Getppid() == 1
+}
+
+func isServerDockerized() bool {
+	_, dockerEnvErr := os.Stat("/.dockerenv")
+	cgroups, _ := ioutil.ReadFile("/proc/self/cgroup")
+	if !os.IsNotExist(dockerEnvErr) || strings.Contains(string(cgroups), "docker") {
+		return true
+	}
+
+	return false
+}
+
+func hideExecShell(cmd *exec.Cmd) {
+
+}
+
+func sendNotification(notificationTitle string, notificationText string) {
+	err := notificator.New(notificator.Options{
+		AppName:     "Stash",
+		DefaultIcon: getIconPath(),
+	}).Push(notificationTitle, notificationText, "", notificator.UR_NORMAL)
```

Comment:
If its only for linux and only used here the included `notificator` library only does  the below and could be omitted
```golang
err := exec.Command("notify-send", "-i", getIconPath(), notificationTitle, notificationText, "-a", "Stash").Run()
```

Reply:
I bounced around the notification libraries a lot while testing, this might be true. I'll take a peek at this

Reply:
It mentions Kdialog in the readme but it was removed in some point, only `notify-send` is used for linux

Reply:
It seems that's correct, I'll just call it directly then and save some vendor changes. I used that library for all three platforms but had to switch after the problems showed up, but it worked on Linux and it stayed.

Reply:
Removed the library, thanks for the tip on this